accounts/ipc.py

```diff
@@ -32,8 +33,8 @@ def authenticated_request(
     happen
     """
 
-    # Access token is expired. Try to refresh access token
-    if user.accesstoken.expires_at < timezone.now():
+    # Access token is non-existent or expired. Try to refresh access token
+    if not user.accesstoken or user.accesstoken.expires_at < timezone.now():
```

Comment:
If the user's access token doesn't exist, then we can pretty much be certain that they don't have a refresh token either and the following line will fail when trying to use the refresh token to get a new access token. Maybe we should just error out or return a 40x if an access token isn't set?